cluster/bin/ansible.sh

```diff
@@ -27,9 +27,9 @@ args=""
 
 if [[ $# -eq 0 ]]; then
   args+="-it "
-  docker create --name "$ctr" -v /var/tmp --entrypoint /usr/local/bin/entrypoint-gcp -e "INSTANCE_PREFIX=${INSTANCE_PREFIX}" $args "${OPENSHIFT_ANSIBLE_IMAGE:-openshift/origin-ansible:latest}" /bin/bash >/dev/null
+  docker create --name "$ctr" -v /var/tmp --entrypoint "/usr/local/bin/entrypoint-${TYPE}" -e "INSTANCE_PREFIX=${INSTANCE_PREFIX}" $args "${OPENSHIFT_ANSIBLE_IMAGE:-openshift/origin-ansible:latest}" /bin/bash >/dev/null
```

Comment:
Putting a quick hold on this, I think we want to fold this in as a new env var to entrypoint rather than continue having the proliferation of entry points.